### PR TITLE
docs: Correct Ubuntu and Debian command line

### PR DESCRIPTION
**Pull request by finikorg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,8 @@ Tested with:
 
 For example, on Ubuntu you can install the dependencies with the following
 command:  
-`apt install cmake libboost-filesystem-dev txt2tags`
+`sudo apt-get install cmake libboost-filesystem-dev txt2tags pkg-config`
+
```
Comment (ybernat) [inline README.md:63]: Oh, why the addition of this empty line, actually?

Comment (finikorg) [reply README.md:63]: This makes output in browser better IMO.

Comment (ybernat) [reply README.md:63]: Because you have removed the double-space at the end of the line, which is the markdown way for line break... :(

Comment (finikorg) [reply README.md:63]: In my browser it is not working, for example: https://github.com/01org/thunderbolt-software-user-space/tree/f02af41870d25587d2c30426c305d24354ef0154

I see following
'''
For example, on Ubuntu you can install the dependencies with the following command:
apt install cmake libboost-filesystem-dev txt2tags On Fedora, use this:
dnf install cmake boost-devel txt2tags
'''

Comment (ybernat) [reply README.md:63]: Which is exactly the commit where you removed the double-space as you can see from the diff there...

See on the commit before it:
https://github.com/01org/thunderbolt-software-user-space/tree/0af08b7ff3659e62dd308058c1223789b748a0c3

**Comment by superm1** [comment]
Any particular reason for switching from apt to apt-get? At least apt has nicer output that shows status.

**Comment by dragon788** [comment]
@superm1 One thing to note is in past versions `apt` output a warning when it doesn't detect an interactive tty something like "apt isn't scripting friendly because the CLI interface isn't stable", but in practical use I've found it seems to multi-thread downloads with nice progress meters and the only thing that will land you in trouble is that `apt upgrade` is equivalent to `apt-get upgrade && apt-get dist-upgrade` so you may end up with a newer kernel than you expected if you aren't careful.

**Comment by superm1** [comment]
IIRC that's only pretty old versions of apt though.  The distributions that adopt a new enough kernel by default to use thunderbolt-software-user-space have modern versions of apt too.

**Pull request merged**